normandy/recipes/tests/test_api.py

```diff
@@ -101,9 +101,15 @@ def test_it_cant_edit_actions_in_use(self, api_client, settings):
 
         res = api_client.patch('/api/v1/action/active/', {'implementation': 'foobar'})
         assert res.status_code == 403
+        assert res.data == {
+            'detail': 'Cannot edit this object while it is in use'
```

Comment:
I've been on the fence about testing specific messages being returned by the API, but it seems like you fall on the "yes, do that" side of that question?